build/commands/scripts/format.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+const path = require('path')
+const fs = require('fs-extra')
+const prettier = require('prettier')
+const program = require('commander')
+
+const config = require('../lib/config')
+const util = require('../lib/util')
+
+program
+  .description(
+    'Format changed code with git cl format and prettier.\n' +
+      'The changed code is the difference between the current files on disk\n' +
+      'and the base branch (origin/master by default).\n' +
+      'This script is also called as part of presubmit checks.'
+  )
+  .option('--base <base branch>', 'set the destination branch for the PR')
+  .option(
+    '--full',
+    'format all lines in changed files instead of only the changed lines'
+  )
+  .option('--presubmit', 'Enables extra presubmit checks in git cl format')
```

Comment:
`'filter out formatters that have dedicated presubmit checks. Used when running the script from a presubmit'`